packages/@ama-sdk/core/cli/update-spec-from-npm.cts

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env node
+
+/*
+ * Update the OpenAPI spec from an NPM package
+ */
+
+import * as minimist from 'minimist';
+import { existsSync } from 'node:fs';
+import { copyFile, readFile } from 'node:fs/promises';
+import type { OpenApiToolsConfiguration } from '../src/fwk/open-api-tools-configuration';
+
+const argv = minimist(process.argv.slice(2));
+const packageName = argv._[0];
+const { help, output, packagePath, quiet } = argv;
+const openApiConfigDefaultPath = './openapitools.json';
+const noop = () => undefined;
+const logger = quiet ? {error: noop, warn: noop, log: noop, info: noop, debug: noop} : console;
+
+if (help) {
+  console.log(`This script can be used to update your local spec file from a given locally installed npm package.
+ Usage: amasdk-update-spec-from-npm <package-name> [--package-path] [--output]
+
+    package-name      The full identifier of the npm package (eg. @my-scope/my-package)
+    --package-path    The relative path inside the npm package where to find the spec file (default: './openapi.yml')
+    --output          The path where the spec file should be copied (default: './openapi.yml')
```

Comment:
This is depending of the format. If we receive a spec in JSON, we want to create a `openapi.json` instead.

Reply:
it's already possible to use JSON file for these parameters, do you mean we should handle `openapi.yml` and `openapi.json` by default when unspecified ?

Reply:
Sorry for the unclear message, yes for the default.
I think to the case I target a JSON file within the `package-path` without providing `output`. I would be quite surprise to retrieve my JSON file content in the `openapi.yml` file :S.